### DISCUSSION TITLE
Video Quality Switch

### DISCUSSION THREAD

**Discussion by Sundaram1911** [cookpete/react-player#1815] [Ideas]
please provide the documentation about the **### video quality switch** for **HLS** Playback in react using react player 
with the provided docs I am able to fetch currentLevel and all Levels using
playerRef.current.getInternalPlayer('hls') - Method
how to set the level and update the currentLevel please Help !!.

**Comment by Tarkik3798** [comment]
Do we have any solution for this issue?

**Comment by sutaungmai** [comment]
Hi guys, I am having the same issue ;) 

Have you guys found the solution for this? :smile:
